test/fixtures/block.json

```diff
@@ -21,6 +21,71 @@
       "expected": "00000000000000000000000000000000000000007fca00000000000000000000"
     }
   ],
+  "zcash": {
```

Comment:
It would perhaps be nice to add some failure cases with a pre-sapling Zcash block.

Reply:
Couldn't find any pre sapling one, but added the Genesis block